clientd3d/d3drender_world.c

```diff
@@ -40,7 +40,7 @@ extern int gNumObjects;
 extern Vector3D sun_vect;
 extern D3DMATRIX view;
 
-static const auto alpha_test_threshold = 128; // Threshold for alpha testing to determine transparency
```

Comment:
Can you add a comment here explaining why 200 is the right number?  It seems super arbitrary.